gnome-initial-setup/pages/endless-eula/gis-endless-eula-page.c

```diff
@@ -42,45 +43,17 @@ typedef struct {
   GtkWidget *metrics_privacy_label;
 
   GisEndlessEulaViewer *eula_viewer;
-  GDBusProxy *metrics_proxy;
 } GisEndlessEulaPagePrivate;
 
 G_DEFINE_TYPE_WITH_PRIVATE (GisEndlessEulaPage, gis_endless_eula_page, GIS_TYPE_PAGE);
 
-static void
-sync_metrics_active_state (GisEndlessEulaPage *page)
-{
-  GisEndlessEulaPagePrivate *priv = gis_endless_eula_page_get_instance_private (page);
-  GError *error = NULL;
-  GtkWidget *widget;
-  gboolean metrics_active;
-
-  widget = priv->metrics_checkbutton;
-  metrics_active = gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (widget));
-
-  if (!priv->metrics_proxy)
-    return;
-
-  g_dbus_proxy_call_sync (priv->metrics_proxy,
-                          "SetEnabled",
-                          g_variant_new ("(b)", metrics_active),
-                          G_DBUS_CALL_FLAGS_NONE, -1,
-                          NULL, &error);
-
-  if (error != NULL)
-    {
-      g_critical ("Unable to set the enabled state of metrics daemon: %s", error->message);
-      g_error_free (error);
-    }
-}
-
 static void
 gis_endless_eula_page_finalize (GObject *object)
 {
   GisEndlessEulaPage *page = GIS_ENDLESS_EULA_PAGE (object);
   GisEndlessEulaPagePrivate *priv = gis_endless_eula_page_get_instance_private (page);
 
-  g_clear_object (&priv->metrics_proxy);
+  g_clear_object (&priv->eula_viewer);
```

Comment:
This is in the wrong commit, but I don't think it's necessary, as mentioned in the other comment.